src/Entity/Enemy.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <flecs.h>
+
+#include "Entity/Components/TransformComponent.h"
+#include "Physics/Components/CollisionComponent.h"
+#include "Physics/Components/VelocityComponent.h"
+#include "Rendering/Components/AnimationStateComponent.h"
+#include "Rendering/Components/SpriteComponent.h"
+
+namespace game
+{
+
+const int ENEMY_WIDTH = 16;
+const int ENEMY_HEIGHT = 16;
+const float ENEMY_SPEED = 150.f;
+
+struct EnemyTag {
+};
+
+class Enemy
+{
+public:
+	static void InitEnemy(flecs::world& ecs);
+
+private:
+	static void EnemyUpdate(flecs::entity enemyEntity, TransformComponent& transformComponent,
+								AnimationStateComponent& animationStateComponent, 
+								CollisionComponent& collisionComponent, VelocityComponent& velocityComponent);
+};
```

Comment:
This is subject to have the class removed since it's all static members.
`EnemyUpdate` would only be in the .cpp.

Reply:
TODO: Fix on refactor